Makefile

```diff
@@ -46,4 +46,4 @@ endif
 	go build -o bin/tidb-dashboard -ldflags '$(LDFLAGS)' -tags "${BUILD_TAGS}" cmd/tidb-dashboard/main.go
 
 run:
-	bin/tidb-dashboard --debug --experimental
+	bin/tidb-dashboard --debug --experimental --allow-non-root-login
```

Comment:
As it is by default, this is no longer needed.

Reply:
Done